vitrina/api/templates/vitrina/api/edp/dcat_ap_rdf.html

```diff
@@ -186,9 +186,15 @@
         {% for keyword in dataset.keywords %}
             <dcat:keyword>{{ keyword }}</dcat:keyword>
         {% endfor %}
+        {% spaceless %}
         {% if dataset.landing_page %}
+        {% if dataset.subclass.name == "information_system" %}
+        <foaf:homepage rdf:resource="{{ dataset.landing_page }}"/>
+        {% else %}
         <dcat:landingPage rdf:resource="{{ dataset.landing_page }}"/>
         {% endif %}
+        {% endif %}
+        {% endspaceless %}
```

Comment:
I know it is out of scope for the whole file, but would be nice to at least add indentations for nested conditions that were added for readability.